src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -646,44 +648,311 @@ private void JSON_response(HttpServletResponse response, String jsonValue,
 			log.error("Missing box integration parameters");
 			return null;
 		}
-		String remoteUserEmail = request.getRemoteUser();
-		
-		if (BoxUtils.getBoxAccessToken() == null)
-		{
+		String remoteUserEmail = request.getRemoteUser() + "@umich.edu";
```

Comment:
Suggestion, make "@umich.edu a constant.
Question, is the email address always going to be @umich.edu? What about friend accounts?